src/cli/init.ts

```diff
@@ -29,34 +23,29 @@ export default class InitCommand extends BaseCommand {
   ]
 
   async run (): Promise<void> {
-    const { flags: originalFlags } = await this.parseWithPrompt(InitCommand)
-    const flags = originalFlags as OutputFlags<typeof InitCommand.flags>
+    await this.initCommand(InitCommand, { serviceRequired: false, db: false })
```

Comment:
Any reason why you use your custom `initCommand` from the `init` supported by oclif?
https://oclif.io/docs/base_class

Reply:
The reason is that i can not i overwrite the init fn interface

Reply:
Hmm because you wanted to call it yourself with the options I assume? 

Well this is I guess alright solution. My idea was to pass these options using constructor something like:

```
constructor() {
	super({ serviceRequired: false, db: false })
}
```

And then the `init()` function is actually called automatically by oclif.

I will leave up to you if you want to rework it to this mechanism.

Reply:
Done